### DISCUSSION TITLE
archive/132772ae-f774-8015-ada6-cd6728c52e15

### DISCUSSION THREAD

**Discussion by giscus** [macjjuni/devlog_latest#23] [Announcements]
# archive/132772ae-f774-8015-ada6-cd6728c52e15

💥사건의 발달 하나의 사이드 프로젝트가 있다. Firebase 무료 호스팅에 배포된 상태인데, 한 통의 메일로 인해 최적화 작업에 들어갔다. 최근 비트코인 가격이 많이 올라 그런

https://kku.dev/archive/132772ae-f774-8015-ada6-cd6728c52e15

**Comment by Siyeop** [comment]
좋은 포스팅 감사합니다.

한 가지 더 개선할 수 있어 보이는 부분이 있어 보여서 댓글 남겨요 ㅎㅎ

btc-cover.webp 이미지가 현재 1MB 인데요.
5950 x 3374 짜리 이미지라서 용량이 큰 것으로 보여요.

그에 비해 이미지가 들어가는 캔버스 영역은 
255 x 127 이라서 불러오는 이미지가 불필요하게 큰 상태로 보입니다.

확대해서 보는 것을 고려해서 가로 세로 x 2 정도로 해서
510 x 254 정도로 리사이징하면 좋을 것 같은데요.

같은 webp 포맷으로 리사이징 해보니 50kb 정도로 조정이 됩니다.
꾸생님의 트래픽에 도움이 되었으면 하는 마음으로 남깁니다 ㅎㅎ

**Reply by macjjuni** [reply]
세심한 피드백 정말 감사드립니다! 😊
덕분에 필요 이상으로 큰 이미지를 쓰고 있었다는 걸 깨달았네요 ㅠㅠ
말씀해 주신 대로 리사이징 해서 더욱 가볍게 최적화했습니당 🚀
앞으로도 종종 피드백 주시면 큰 도움이 될 것 같아요 ㅎㅎ
